corehq/apps/smsbillables/models.py

```diff
@@ -365,3 +376,39 @@ def _get_usage_fee(cls, domain, direction):
                 % (direction, domain)
             )
         return usage_fee
+
+    @classmethod
+    def _get_twilio_charges(cls, backend_instance, backend_message_id, direction, couch_id):
+        def _get_twilio_client(twilio_backend_id):
+            twilio_backend = SQLMobileBackend.load(
+                twilio_backend_id,
+                api_id=SQLTwilioBackend.get_api_id(),
+                is_couch_id=True,
+                include_deleted=True,
+            )
+            config = twilio_backend.config
+            return TwilioRestClient(config.account_sid, config.auth_token)
```

Comment:
it looks like this is going to lookup the fee even for twilio backends that are owned by domains (i.e., is_global=False).  do we need to handle that here, or is there elsewhere in the framework that ignores charging fees for domain-owned backends?